packages/common/utils/src/delay.spec.ts

```diff
@@ -0,0 +1,20 @@
+import { delay } from './delay';
+
+describe('delay', () => {
+  beforeEach(() => {
+    jest.useFakeTimers();
+  });
+
+  afterEach(() => {
+    jest.useRealTimers();
+  });
+
+  it('should return a promise which is resolved after given milliseconds', () => {
+    const milliseconds = 3000;
```

Comment:
```suggestion
    const DELAYING_TIME = 3000;
```

I hope the role of the variable is clear

Reply:
I agree with you. I think it's better to name a constant which means a role. I fix [it](https://github.com/toss/slash/pull/388/commits/37f1ccdd0cd47429cf2b93dc5120ec86fcd8c954).